meta/configure_vlan_spec.yaml

```diff
@@ -0,0 +1,18 @@
+---
+argument_spec:
+  ansible_network_os:
+    description:
+    - Set the name of the Ansible network OS platform.  This value should be
+      set to `junos` for this provider.
+    required: yes
+
+  id:
+    description:
+    - Provide the vlan name that need to be configured to the device
```

Comment:
Is this really vlan name? or should it say provide the vlan ID that needs to be configured...